test-network-nano-bash/network.sh

```diff
@@ -52,22 +52,40 @@ networkStart() {
   # shellcheck disable=SC2064
   trap networkStop 0 1 2 3 15
 
+  if [ "$ORDERER_TYPE" = "BFT" ]; then
+    yq -i '.Capabilities.Channel.V3_0 = true | (... | select(tag == "!!merge")) tag = ""' configtx.yaml
+    yq -i '.Capabilities.Channel.V2_0 = false | (... | select(tag == "!!merge")) tag = ""' configtx.yaml
+
+    yq -i 'del(.Consensus.SnapDir)' ../config/orderer.yaml
```

Comment:
In test-network-nano-bash we leave orderer.yaml as-is, and override values using environment variables.